CMakeLists.txt

```diff
@@ -41,10 +40,11 @@ catkin_package(
 
 # include_directories(include)
 include_directories( 
-    ${CMAKE_SOURCE_DIR}/${PROJECT_NAME}
+    include
+  SYSTEM
```

Comment:
Alignment

Reply:
this is correct as we group packages after the scope specifier